packaging/ubuntu-16.04/rules

```diff
@@ -166,6 +166,8 @@ override_dh_auto_build:
 	# this is the main go build
 	SNAPD_VANILLA_GO=$$(which go) PATH="$$(pwd)/packaging/build-tools/:$$PATH" dh_auto_build -- $(BUILDFLAGS) $(TAGS) $(GCCGOFLAGS)
 
+	(cd _build/bin && GOPATH=$$(pwd)/.. GOCACHE=/tmp/go-build CGO_ENABLED=0 go build $(BUILDFLAGS) $(GCCGOFLAGS) -tags nomanagers $(DH_GOPKG)/cmd/snap)
```

Comment:
Can you please replicate this across the packaging helpers?

Reply:
What helpers do you mean? Not snapd.mk?

Reply:
This should probably use $(TAGS) instead because go will just act strangely if you pass multiple `-tags` lines, i.e. it needs very specific syntax: `-tags "tag1 tag2"`. The packaging/debian-sid should give an inspiration how this can be done.

Reply:
given we are doing these changes, we should probably do them for snap-bootstrap as well already, and also add it to the test, that's only relevant for the distros where we build it though

Reply:
Added snap-bootstrap to the spread test check. Tinkering with debian rules for it was producing confusing errors about cgo or some imports though, I need investigate it more.

Reply:
mmh @mvo5 can confirm, but I don't think we care about snap-bootstrap on debian, it was just probably easier to build it than not build it, if it gives problems we can probably just stop building it there by removing the code first or something

Reply:
with snap-bootstrap here I mean the executable, not the boostrap lib, we need to build the dummy version of it on debian